stablehlo/dialect/VhloTypes.h

```diff
@@ -0,0 +1,67 @@
+/* Copyright 2023 The StableHLO Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef STABLEHLO_TRANSFORMS_VHLOTYPES_H
+#define STABLEHLO_TRANSFORMS_VHLOTYPES_H
+
+#include "mlir/Transforms/DialectConversion.h"
+#include "stablehlo/dialect/Version.h"
+#include "stablehlo/dialect/VhloDialect.h"
+
+namespace mlir {
+namespace vhlo {
+
+class VersionedTypeConverterBase : public TypeConverter {
```

Comment:
Wdyt about renaming VersionedTypeConverterBase to VhloTypeConverterBase? The original name is probably a remnant from the times when the dialect was called VersionedHLO?